### ISSUE TITLE
Web accessibility

### ISSUE THREAD

**Issue by zaccolley** [codebar/planner#505]
There are places where the colour contrast is a bit low which fails some accessibility guidelines set by the W3C.

e.g:

![image](https://cloud.githubusercontent.com/assets/2591901/21057123/34aa31f0-be30-11e6-8f89-c4c45b28777d.png)
 This has a contrast ratio of 2.67.

Also seems to be some stuff we could improve for assisitive tech:
https://tenon.io/testNow.php?url=http://codebar.io/

Should we try make the site compliant to the WCAG? https://www.w3.org/WAI/intro/wcag

**Comment by BNewing** [comment]
@alexbokii and I are keen to give this a go in a few weeks when we're back in the same place :)

**Comment by matyikriszta** [comment]
@zaccolley it would be good to write a list of what needs to be done to make the site more accessible, for example the markup could use some work too to make it more semantic

**Comment by zaccolley** [comment]
Forgot to reply to this. I'll make some more concrete issues yeah.

**Comment by zaccolley** [comment]
I'm gonna close this as I've made specific issues
